x/stakeibc/keeper/msg_server_restore_interchain_account_test.go

```diff
@@ -172,32 +245,42 @@ func (s *KeeperTestSuite) VerifyHostZoneUnbondingStatus(expectedUnbondingRecords
 	}
 }
 
+// Helper function to check that each LSMTokenDepoit's status was either left alone or reverted to it's prior status
+func (s *KeeperTestSuite) verifyLSMDepositStatus(expectedLSMDeposits []LSMTokenDepositStatusUpdate, revert bool) {
+	for _, expectedLSMDeposit := range expectedLSMDeposits {
+		actualLSMDeposit, found := s.App.StakeibcKeeper.GetLSMTokenDeposit(s.Ctx, expectedLSMDeposit.chainId, expectedLSMDeposit.denom)
+		s.Require().True(found, "lsm deposit found")
+
+		// Only revert record if the revert option is passed and the host zone matches
+		expectedStatus := expectedLSMDeposit.initialStatus
+		if revert && actualLSMDeposit.ChainId == HostChainId {
+			expectedStatus = expectedLSMDeposit.revertedStatus
+		}
+		s.Require().Equal(expectedStatus.String(), actualLSMDeposit.Status.String(), "lsm deposit %d")
```

Comment:
what integer was the %d for? The index of the expectedLSMDeposit?

Reply:
ah yeah good catch. fixed